src/test/java/uk/gov/companieshouse/certificates/orders/api/controller/CertificateItemsControllerIntegrationTest.java

```diff
@@ -65,12 +93,15 @@
     @MockBean
     private EtagGeneratorService etagGenerator;
 
+    @MockBean
+    private IdGeneratorService idGeneratorService;
+
     @MockBean
     private CompanyService companyService;
 
     private static final String CERTIFICATES_URL = "/orderable/certificates/";
-    private static final String EXPECTED_ITEM_ID = "CHS00000000000000001";
-    private static final String UPDATED_ITEM_ID  = "CHS00000000000000002";
+    private static final String EXPECTED_ITEM_ID = "CRT-123456-123456";
+    private static final String UPDATED_ITEM_ID  = "ORD-123456-123457";
```

Comment:
Probably should make this `CRT-...` too.

Reply:
Thanks, updated